docs/docs/reference/attestations.md

```diff
@@ -0,0 +1,193 @@
+---
+sidebar_position: 1
```

Comment:
Hmm no big reason just that this section only has this article and the operators have multiple and thought it would look nice if you expand the operator one to have this on top so you don't lose it.